docker-run-generate-outputs.sh

```diff
@@ -23,16 +23,16 @@ while [[ $# -gt 0 ]]; do
 done
 
 # Check that the correct number of arguments were provided.
-if [[ $# -ne 12 ]]; then
+if [[ $# -ne 14 ]]; then
     echo "Usage: ./docker-run-generate-outputs.sh
     [--profile-cpu <cpu-profile-output-path>] [--profile-memory <memory-profile-output-path>]
     <user> <google-cloud-credentials-file-path> <pipeline-configuration-file-path>
     <raw-data-dir> <prev-coded-dir> <messages-json-output-path> <individual-json-output-path> <icr-output-dir>
-    <coded-output-dir> <messages-output-csv> <individuals-output-csv> <production-output-csv>"
+    <coded-output-dir> <messages-output-csv> <individuals-output-csv> <production-output-csv>
```

Comment:
This is now a bit ugly because it means you have to pass paths to the messages, individuals csvs/traced data etc. even if you don't generate them.

Given that we don't run this script directly this is really not an urgent problem, but please at least make an issue for it.

Reply:
That's true. Added an issue on this https://github.com/AfricasVoices/Project-IMAQAL/issues/87. Thanks.